tests/assertions.ts

```diff
@@ -43,7 +43,7 @@ export async function assertGeneratedCorrectly({
     `The test app has a (dev)dependency on the addon`
   ).to.include(addonName);
 
-  let expectedStaticFiles = ['README.md', 'LICENSE.md'];
```

Comment:
what happened to LICENSE.md?

Reply:
Nothing really, I was just a bit lazy for this test tbh.

The thing is that I use this helper function also for the new monorepo tests. But the License file is not generated in the monorepo case (it's a root level file). So by having this function only assert for files that are expected to exist in the actual addon (sub)folder (which are for example `README.md` and `CONTRIBUTING.MD`, but _not_ `LICENSE.md`), we can reuse it for both scenarios. Otherwise we would have to have two different helper functions, or introduce some conditionality...

Reply:
Btw, `README.md` and `CONTRIBUTING.MD` only work here - also in the no existing monorepo case - because they are _copied_ into the addon folder, either by rollup (no existing monorepo) or by the blueprint (existing monorepo), while in our blueprint repo they only exist at the root level. That's a bit confusing at first if you don't remember that, I had to remind myself of this fact also...